Tests/js/models/extensions/assets/ez-contentinfo-base-tests.js

```diff
@@ -167,6 +167,29 @@ YUI.add('ez-contentinfo-base-tests', function (Y) {
 
             Assert.isTrue(callbackCalled, 'Should call callback function');
         },
+
+        'Should handle load versions by status without versions': function () {
```

Comment:
that's not clear, actually you fixed the error handling, so the test name should probably contain 'error handling' :)